subprojects/android-test/test-instrumentation-runner/src/main/kotlin/com/avito/android/runner/OrchestratorDelegate.kt

```diff
@@ -0,0 +1,10 @@
+package com.avito.android.runner
+
+import android.os.Bundle
+
+interface OrchestratorDelegate {
```

Comment:
I don't understand the difference in responsibility between this abstraction and `InstrumentationDelegate`.
Would you mind explaining that?

Reply:
i just delegated logic around orchestrator from InstrumentationTestRunner code
InstrumentationDelegate is a logic delegated depending on orchestrator argument, that's why its separated

Reply:
What I see:
`test-instrumentation-runner` module doesn't know about avito test runner.
`InstrumentationDelegate` is in this module. It is a common delegate for `AndroidJUnitRunner`
`OrchestratorDelegate` is also in this module.
I didn't get how to decide which delegate to use when I will need to extract something else.

Is it correct that `OrchestratorDelegate` should encapsulate knowledge about AndroidX test orchestrator?

Reply:
yes; i see it like this

Reply:
Ok, maybe naming it as `AndroidXOrchestratorDelegate` will show it more obviously.
It took time to realize it because "orchestration" for me is a general term.